test/lolex-test.js

```diff
@@ -2053,6 +2053,17 @@ describe("lolex", function () {
                 clock.runAll();
                 assert(!called);
             });
+
+            it("passes arguments when installed - GitHub#122", function () {
+                var clock = this.clock = lolex.install();
```

Comment:
Oh, I meant to do that and use it to uninstall - and then uninstalled myself. Fixing.